itou/users/tests/test_models.py

```diff
@@ -1442,3 +1436,33 @@ def test_save_creates_a_job_seeker_profile(user_kind, profile_expected):
     user = User(kind=user_kind)
     user.save()
     assert hasattr(user, "jobseeker_profile") == profile_expected
+
+
+def test_save_erases_id_national_pe_if_details_change():
+    user = UserFactory(kind=UserKind.JOB_SEEKER)
+
+    def reset_profile_and_save_user():
+        user.jobseeker_profile.id_national_pe = "1234567890123"
```

Comment:
On ne voudrait pas mettre un:
`user.jobseeker_profile.pe_certified_at = timezone.now()` ici ?
Car sinon, j'ai l'impression que les `assert user.jobseeker_profile.pe_certified_at is None` vérifient juste que cette valeur reste à `None` tout le long du test

Reply:
En effet.